mrp/examples/14_habitat/README.md

```diff
@@ -0,0 +1,12 @@
+# Example 14: Habitat
+
+In this example we run the habitat viewer example.
+
+To launch:
+```sh
+mrp up
+```
+
+No need to down mrp. Closing the UI is enough.
```

Comment:
Either "No need to down MRP" or "No need to `mrp down`"?

Reply:
Done.